src/test/scala/QuartzTypedSchedulerFunctionalSpec.scala

```diff
@@ -0,0 +1,459 @@
+      val probe = testKit.createTestProbe[AnyRef]()
+      receiver ! NewProbe(probe.ref)
+
+      val jobDt = `extension`.createTypedJobSchedule(toDeleteSheduleJobName, receiver, Tick, Some("Creating new dynamic schedule for deleteJobSchedule test"), "*/7 * * ? * *")
+
+      noException should be thrownBy {
+        // Delete existing scheduled job
+        val success = `extension`.deleteJobSchedule(toDeleteSheduleJobName)
+        if (success) {
+
+          // Create a new schedule job reusing former toDeleteSheduleJobName. This will fail if delebeJobSchedule is not effective.
+          val newJobDt = `extension`.createTypedJobSchedule(toDeleteSheduleJobName, receiver, Tick, Some("Creating new dynamic schedule after deleteJobSchedule success"), "8 * * ? * *")
+          val jobCalender = Calendar.getInstance()
+          jobCalender.setTime(newJobDt)
+          jobCalender.get(Calendar.SECOND) mustEqual 8
+        } else {
+          fail(s"deleteJobSchedule(${toDeleteSheduleJobName}) expected to return true returned false.")
+        }
+      }
+    }
+
+    "Delete a non existing job schedule Cron Job with no error and a return value false" in {
+
+      val nonExistingCronToBeDeleted = "nonExistingCronToBeDeleted"
+
+      val receiver = testKit.spawn(ScheduleTestReceiver())
+      val probe = testKit.createTestProbe[AnyRef]()
+      receiver ! NewProbe(probe.ref)
+
+      noException should be thrownBy {
+        // Deleting non existing scheduled job
+        val success = `extension`.deleteJobSchedule(nonExistingCronToBeDeleted)
+        // must return false
+        if (success) {
+          fail(s"deleteJobSchedule(${nonExistingCronToBeDeleted}) expected to return false returned true.")
+        }
+      }
+    }
+
+  }
+
+  sealed trait QuartzMessage
+  case class NewProbe(probe: ActorRef[QuartzMessage]) extends QuartzMessage
+  case object Tick extends QuartzMessage
+  case object Tock extends QuartzMessage
+  case class TockWithFireTime(scheduledFireTime: Long, previousFireTime: Option[Long], nextFireTime: Option[Long]) extends QuartzMessage
+
+  object ScheduleTestReceiver {
+    private val log: Logger = Logger.getLogger(getClass.getSimpleName)
+    var probe: ActorRef[QuartzMessage] = _
+
+    def apply(): Behavior[AnyRef] = Behaviors.receiveMessage {
+      case NewProbe(_p) =>
+        probe = _p
+        Behaviors.same
+
+      case Tick =>
+        log.info(s"Got a Tick.")
+        probe ! Tock
+        Behaviors.same
+
+      case MessageWithFireTime(Tick, scheduledFireTime, previousFireTime, nextFireTime) =>
+        log.info(s"Got a Tick for scheduledFireTime=${scheduledFireTime} previousFireTime=${previousFireTime} nextFireTime=${nextFireTime}")
+        probe ! TockWithFireTime(
+          scheduledFireTime.getTime,
+          previousFireTime.map(u => u.getTime),
+          nextFireTime.map(u => u.getTime))
+        Behaviors.same
+    }
+  }
+
+}
+
+object SchedulingTypedFunctionalTest {
```

Comment:
This object appears to be completely duplicated between the two test packages, no? Can you move it out to its own file and update both test classes to point to it?

Reply:
Fixed ✅